include/arc_utilities/arc_helpers.hpp

```diff
@@ -586,6 +588,63 @@ namespace arc_helpers
         }
     }
 
+    template<typename ColorType>
+    inline ColorType Multiply(const ColorType& color, const float factor)
```

Comment:
When would a `Multiply` function be useful for colors?

Reply:
When you want to make something darker or lighter for example. I.e. `0.5 * blue` gives you a darker blue.